CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+# Für Entwickler
+
+## Erstellen von Windows builds
+Stelle sicher, dass alle Pakete aus der requirements.txt installiert sind. Aktuell nutzen wir außerdem Python 3.11 und cx_Freeze 7.0.0 zum Erstellen von Windows builds. Nun solltest du build.bat im Sephrasto root ausführen können.
```

Comment:
Solche zusätzlichen requirements für Entwickler packt man am besten in eine "requirements-dev-windows.txt" oder dergleichen.